client/src/app/pages/dashboard/components/task-item/task-item-in-progress/task-item-in-progress.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { EditTask, Task } from '../../../models/models.tasks';
+import { MatDialog } from '@angular/material/dialog';
+import { EditTaskComponent } from 'src/app/pages/dialogs/edit-task/edit-task.component';
+import { DeleteTaskComponent } from 'src/app/pages/dialogs/delete-task/delete-task.component';
+
```

Comment:
Again name is very strange

Reply:
_task-item-in-progress_ has a checkbox while _task-item-completed_ does not. This is why I had to create two different components and name them differently.